terra_layer/style.py

```diff
@@ -386,27 +452,49 @@ def gen_legend_circle(min, max, size, color):
         for b in boundaries
     ]
 
+    if include_no_value:
+        ret += [
```

Comment:
`ret.append` instead ?

Reply:
As you wish, Nahuell tell me .append() is too java-ish for python.

Reply:
Fixed.